automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ProxyRedirectClient.scala

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import akka.Done
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.server.Directives.{get => httpGet, _}
+import akka.http.scaladsl.server.Route
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.workbench.ResourceFile
+import org.broadinstitute.dsde.workbench.service.RestClient
+
+import scala.concurrent.Future
+import scala.io.Source
+
+object ProxyRedirectClient extends RestClient with LazyLogging {
+  // Note: change to "localhost" if running tests locally
+  val host = java.net.InetAddress.getLocalHost.getHostName
```

Comment:
Important to note, running in Jenkins requires the hostname as `java.net.InetAddress.getLocalHost.getHostName`. 

I think this is because the automation tests run in a docker container in Jenkins, and this is how the docker network is configured. Could probably configure the docker container to work with "localhost", but that is a little scary to touch.